apps/storefront/src/pages/Login/index.tsx

```diff
@@ -126,7 +98,7 @@ export default function Login(props: PageProps) {
   };
 
   useEffect(() => {
-    const logout = async () => {
+    const asyncLogout = async () => {
```

Comment:
still grim, but maybe we can swap this immediate definition and invocation with an async IIFE?

```tsx
  useEffect(() => {
    (async () => {
       // stuff...
    })();
  }, [b3Lang, endMasquerade, isLoggedIn, isAgenting, searchParams, selectCompanyHierarchyId]);
```

otherwise we have an `asyncLogout` and a `logout` (which is also async)